string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,22 @@
+/* exported capitalizeWord */
+/*
+- create storage for output
+- make all characters upper case
+- go through each character one by one
+- starting at the second character, make all characters lower case
+- add each character to storage
+- return output
+ */
+
+function capitalizeWord(word) {
+  var output = '';
+  output = word[0].toUpperCase(word[0]);
```

Comment:
`toUpperCase` and `toLowerCase` do not have any arguments, you can remove the values between the parentheses